### PR TITLE
[JENKINS-53842] Upgrade Maven version used for Blue Ocean CI

### PR DESCRIPTION
**Pull request by olamy**
# Description

See [JENKINS-53842](https://issues.jenkins-ci.org/browse/JENKINS-53842).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -35,11 +35,14 @@ node() {
...
-          try {
-            sh "mvn clean install -B -DcleanNode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.test.failure.ignore -s settings.xml -Dmaven.artifact.threads=30"
-          } catch(e) {
-            throw e;
+          timeout(time: 90, unit: 'MINUTES') {
+            try {
+              sh "mvn clean install -V -B -DcleanNode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.test.failure.ignore -s settings.xml -Dmaven.artifact.threads=30"
+            } catch(e) {
+              throw e;
+            }
```
Comment (batmat) [inline Jenkinsfile]: This try/catch looks like a no-op. Can you clarify the intent here?

Comment (batmat) [reply Jenkinsfile]: My bad. It still looks fishy, but I see you only added the timeout

**Comment by olamy** [comment]
@batmat agree so I remove this weird code.

File: Jenkinsfile [hunk 2]
```diff
@@ -35,11 +35,10 @@ node() {
         }
 
         stage('Building BlueOcean') {
-          try {
-            sh "mvn clean install -B -DcleanNode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.test.failure.ignore -s settings.xml -Dmaven.artifact.threads=30"
-          } catch(e) {
-            throw e;
+          timeout(time: 90, unit: 'MINUTES') {
```
Comment (michaelneale) [inline Jenkinsfile]: Nice

Comment (michaelneale) [reply Jenkinsfile]: *self loathing over never thinking of this intensifies*

**Pull request merged**